src/applications/ezr/config/chapters/veteranInformation/emergencyContacts.js

```diff
@@ -0,0 +1,181 @@
+import merge from 'lodash/merge';
+import ezrSchema from 'vets-json-schema/dist/10-10EZR-schema.json';
+import { arrayBuilderPages } from '~/platform/forms-system/src/js/patterns/array-builder';
+import {
+  arrayBuilderYesNoSchema,
+  arrayBuilderYesNoUI,
+  addressUI,
+  addressSchema,
+  fullNameUI,
+  phoneUI,
+  selectUI,
+  titleUI,
+  yesNoUI,
+  yesNoSchema,
+} from '~/platform/forms-system/src/js/web-component-patterns';
+import content from '../../../locales/en/content.json';
+import { isEmergencyContactsEnabled } from '../../../utils/helpers/form-config';
+
+const {
+  veteranContacts: { items: contact },
+} = ezrSchema.properties;
+
+const { fullName, primaryPhone, relationship, address } = contact.properties;
+
+const arrayBuilderOptions = {
+  arrayPath: 'veteranContacts',
+  nounSingular: 'emergency contact',
+  nounPlural: 'emergency contacts',
+  required: false,
+  maxItems: 2,
+  isItemIncomplete: item =>
+    !item?.fullName?.first ||
+    !item?.fullName?.last ||
+    !item?.primaryPhone ||
+    !item?.relationship,
+  text: {
+    getItemName: item => {
+      if (item && item.fullName && item.fullName.first && item.fullName.last) {
```

Comment:
Would it be a good idea to pull some of these out to functions that could be reused? I see we have the same logic below for the deleteDescription. I know sometimes we have put stuff in a `utils/validations.js` file, although if we are only using them on this page we could probably just put them at the bottom?

Reply:
To piggyback on this... there is a high chance these methods will need broken out and imported in so you can get unit test coverage.